### PR TITLE
Add Geonear instructions to ReadMe. Closes #1878

### LINKED ISSUE #1878: Please Add 'distanceField' in mongodb geoNear query
I'm trying to make a geoNear query:
```
            '$geometry' => [
                'type' => 'Point',
                'coordinates' => [
                    $longitude,
                    $latitude
                ],
            ],
            '$maxDistance' => 2000,
            'distanceField' => 'distance'
```

and getting an error "invalid argument in geo near query: distanceField"

But MongoDB supports 'distanceField' option. Reference https://docs.mongodb.com/manual/reference/operator/aggregation/geoNear

Please add 'distanceField' in geo near query, this will make this library better !!
And thanks for this amazing library :+1:

### PR DESCRIPTION
**Pull request by henriquetroiano**
Hey guys!

Here it is a PR with the geoNear instructions.

That closes #1878

### CODE REVIEW

**Review by Smolevich — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1046,17 +1083,19 @@ Schema::create('users', function ($collection) {
...
 MongoDB specific operations:
-- background
-- sparse
-- expire
-- geospatial
+
+-   background
+-   sparse
+-   expire
+-   geospatial
```
Comment (divine) [inline README.md:1098]:
```suggestion

-   background
-   sparse
-   expire
-   geospatial
```

Revert this back.

File: README.md [hunk 2]
```diff
@@ -1046,17 +1083,19 @@ Schema::create('users', function ($collection) {
...
-- collection
-- hasCollection
-- index and dropIndex (compound indexes supported as well)
-- unique
+
+-   create and drop
+-   collection
+-   hasCollection
+-   index and dropIndex (compound indexes supported as well)
+-   unique
```
Comment (divine) [inline README.md:1091]: Revert this back.

File: README.md [hunk 3]
```diff
@@ -734,20 +770,21 @@ DB::collection('users')
...
 The only available relationships are:
- - hasOne
- - hasMany
- - belongsTo
- - belongsToMany
+
+-   hasOne
+-   hasMany
+-   belongsTo
+-   belongsToMany
```
Comment (divine) [inline README.md:782]:
```suggestion

-   hasOne
-   hasMany
-   belongsTo
-   belongsToMany
```

Revert this back.

**Review by divine — APPROVED** [review]
Thanks!

**Pull request merged**